contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -425,7 +425,13 @@ CAST(Ext.default_schema_name AS SYS.SYSNAME) AS default_schema_name,
 CAST(Ext.create_date AS SYS.DATETIME) AS create_date,
 CAST(Ext.modify_date AS SYS.DATETIME) AS modify_date,
 CAST(Ext.owning_principal_id AS INT) AS owning_principal_id,
-CAST(CAST(Base2.oid AS INT) AS SYS.VARBINARY(85)) AS SID,
+CAST(
+  CASE 
+    WHEN Ext.orig_username = 'dbo' THEN CAST(Base3.oid AS INT)
+    WHEN Ext.orig_username = 'guest' THEN 0
+    ELSE CAST(Base2.oid AS INT)
+  END 
+AS SYS.VARBINARY(85)) AS SID,
```

Comment:
We show same value for sid also in user_token view. Incorporate this changes in that view as well.

sys.user_token view should also have the similar behaviour.

SQL server
```
-- On master - 
1> select suser_sname(sid) from sys.user_token where name = 'dbo';
2> go
 
-
sa

(1 rows affected)

-- On new_db (different owner - l2)
[abanju@dev-dsk-abanju-1c-ea0b53a0]~/oss% tsqlcon -U l2 -d tk
1> select suser_sname(sid) from sys.user_token where name = 'dbo';
2> go
 
-
l2

(1 rows
```